spec/controllers/v0/hca_attachments_controller_spec.rb

```diff
@@ -11,5 +11,12 @@
 
       expect(JSON.parse(response.body)['data']['attributes']['guid']).to eq HcaAttachment.last.guid
```

Comment:
I think creating an additional expectation that the attachment file stored in the DB matches the one we uploaded? `pdf_fill/extras.pdf` in this case.

What came around to bite us last time, was the lack of happy path testing. That empty hash was getting stored to the database but we were none the wiser. We should verify that the file uploaded is the one that's stored.